doc/lightwallet.md

```diff
@@ -0,0 +1,186 @@
+Light Wallet RingCT
+====
+
+Setup
+---------------------
+To setup a light wallet for RingCT transactions we must have two parts. 
+
+1. A watchonly server setup and running, this server acts as a full Veil node syncing blocks and handling transaction information.
+   1. To run a server like this. All you need to do is run Veild with the flag -watchonly=1 placed in the veil.conf
+   2. Then you can setup a api server to act as the passthrough to the veild via rpc commands
+2. On your local desktop, you must have the ability to launch veild with the flag -lightwallet=1. This flag makes veild act like a light wallet. It will not connect to peers, it will not sync the blockchain. 
+   1. This is used to compute ringct transactions, to be broadcasted later via the watchonly server.
+---------------------------   
+## Prerequisites
+1. You have a ringct wallet that has a balance
+2. You have the ability to generate the following keys for the given address
+   1. Spend Secret Key
```

Comment:
Needs instructions on how to generate these. Or they should be retrievable in the lightwallet itself so the user doesn't have to grab all these themselves (ideally the commands shouldn't have to prompt for them in that case, only check that the wallet is unlocked).